compat/class-instant-articles-jetpack.php

```diff
@@ -39,6 +39,11 @@ private function _fix_youtube_embed() {
 	 */
 	private function _fix_facebook_embed() {
 
+		// Don't try to fix facebook embeds unless we're in Instant Articles context
+		// to prevent mangled output on frontend
+		if ( !doing_action( 'instant_articles_before_transform_post' ) )
+			return;
```

Comment:
I'm not sure how to do it via github, but there are some Coding Standards that need fixing:

```
// Don't try to fix facebook embeds unless we're in Instant Articles context.
// This prevents mangled output on frontend.
if ( ! doing_action( 'instant_articles_before_transform_post' ) ) {
    return;
}
```